src/strategies/ve-balance-of-at/index.ts

```diff
@@ -21,10 +21,16 @@ export async function strategy(
     typeof snapshot === 'number' ? snapshot : await provider.getBlockNumber();
 
   const multi = new Multicaller(network, provider, abi);
-  addresses.forEach((address) =>
-    multi.call(address, options.address, 'balanceOfAt', [address, blockTag])
-  );
-  const result: Record<string, BigNumberish> = await multi.execute();
+  let result: Record<string, BigNumberish> = {};
+
+  const chunkSize = 200;
+  for (let i = 0; i < addresses.length; i += chunkSize) {
+    const chunk = addresses.slice(i, i + chunkSize);
+    chunk.forEach((address) =>
+      multi.call(address, options.address, 'balanceOfAt', [address, blockTag])
+    );
+    result = await multi.execute();
+  }
```

Comment:
Hey @spicysquid168 Maybe try using the `pagination` strategy, It is not recommended to add pagination with 200 addresses inside the strategy

Reply:
Hey @ChaituVR, thanks for the suggestion. We will go with `pagination` strategy 👍